hashintel/hash
# Make edit bar on entity-editor / entity-type-editor sticky

## 🌟 What is the purpose of this PR?

This PR makes edit bar component on entity editor and entity type editor `position: sticky` to make it stick to the top when user scrolls down to improve UX. Otherwise it's easy for users to miss that edit bar appeared at the top of the page, if they scrolled down.

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203543021352032/1203718933782050/f) _(internal)_

## 🔍 What does this change?

- Add `sx` support to `top-context-bar`

## ❓ How to test this?

1.  Checkout the branch / view the deployment
1.  Confirm that edit bar on entity-editor and entity-type-editor is sticking to top when page is scrolled down

## 📹 Demo
 


https://user-images.githubusercontent.com/39553853/212683054-36f2dfd9-31d9-4d3b-98ab-6acb4181d0fb.mov
